src/settingStorage.js

```diff
@@ -0,0 +1,26 @@
+import AsyncStorage from '@react-native-community/async-storage';
```

Comment:
Please add this filer under service folder

Reply:
I have deleted this file as it was not needed